test/ecto/changeset_test.exs

```diff
@@ -723,6 +724,15 @@ defmodule Ecto.ChangesetTest do
     assert changeset.changes == %{}
     assert changeset.errors == [title: {"is blank", [validation: :required]}, body: {"is blank", [validation: :required]}]
 
+    # When :trim option is false
```

Comment:
Sorry for the off-topic, but do you guys think it could be better to put these commens in different tests so in case of errors it's easier to know what's failing?

```elixir
# Option 1
describe "validate_required/2" do
  test "when valid" do
    # ...
  end

  test "when missing" do
    # ...
  end

  test "when :trim option is false" do
    # ...
  end
end

# Option 2
test "validate_required/2 when valid" do
  # ...
end

test "validate_required/2 when missing" do
  # ...
end

test "validate_required/2 when :trim option is false" do
  # ...
end
```

If you like any of the options I could apply those changes in a different PR :)

Reply:
I like option 1 more but this should be decided by someone which is more involved in the project like @michalmuskala or @josevalim

Reply:
@fertapric a PR that moves to this style would be appreciated. This project started way before Elixir v1.0, so we didn't have such constructs at the time. :)